data/jsons/dependencies.json

```diff
@@ -1,4 +1,93 @@
 {
+    "definitions" : {
+        "VERSION_MAJOR": 0,
+        "VERSION_MINOR": 0,
+        "VERSION_RELEASE": 1,
+        "VERSION": "{VERSION_MAJOR}.{VERSION_MINOR}.{VERSION_RELEASE}",
+        "DESTDIR": "/",
+        "SYSCONF": "/etc/",
+        "LIBDIR": "{PREFIX}/lib/",
+        "DATADIR": "{PREFIX}/share/",
+        "BINDIR": "{PREFIX}/bin/",
+        "INCLUDEDIR": "{PREFIX}/include/",
+        "SOL_DATADIR": "{DATADIR}soletta/",
+        "SOL_INCLUDEDIR": "{INCLUDEDIR}soletta/",
+        "MODULESDIR": "{LIBDIR}soletta/modules/",
+        "SOL_FLOW_DATADIR": "{SOL_DATADIR}flow/",
+        "DESCDIR": "{SOL_FLOW_DATADIR}descriptions/",
+        "SCHEMADIR": "{SOL_FLOW_DATADIR}schema/",
+        "PCDIR": "{LIBDIR}pkgconfig/",
+        "PKGNAME": "soletta",
+        "PKGSYSCONFDIR": "{SYSCONF}",
+        "FLOWMODULESDIR": "{MODULESDIR}flow/",
+        "PINMUXDIR": "{MODULESDIR}pin-mux/",
+        "LINUXMICROMODULESDIR": "{MODULESDIR}linux-micro/"
+    },
```

Comment:
Since we are doing this here... I've seen in several places that we end up with paths formed with // sometimes, and to me, "{SOME_VAR}more/path/" looks a lot more like a typo than "{SOME_VAR}/more/path".